pkg/data/components/StartSampling.yaml

```diff
@@ -1,4 +1,4 @@
-kind: StartSampling
+kind: SamplingRuleHandler
```

Comment:
Of all of these, this is the only one I'm less than happy with. 

How about SamplingSequencer? @verythorough ?

I put a note in the doc too.

Reply:
Updated